src/test/java/com/epam/test_generator/services/SuitServiceTest.java

```diff
@@ -139,4 +146,83 @@ public void removeSuitTest_expectNotFoundException() {
         suitService.removeSuit(SIMPLE_SUIT_ID);
     }
 
+    @Test
+    public void updateSuitRowNumber() {
+        SuitRowNumberUpdate srnu1 = new SuitRowNumberUpdate();
+        srnu1.setId(5L);
+        srnu1.setRowNumber(2);
+
+        SuitRowNumberUpdate srnu2 = new SuitRowNumberUpdate();
+        srnu2.setId(4L);
+        srnu2.setRowNumber(1);
+
+        List<SuitRowNumberUpdate> rowNumbers = new ArrayList<>(
+                Arrays.asList(srnu1, srnu2)
+        );
+
+        final List<SuitRowNumberUpdate> sortedRowNumbers = rowNumbers
```

Comment:
Duplication of SuitService code in unit tests is a sign of not-well designed SuitService. Try to design code is a way it does not require duplication in test cases.